bin/hdfcoinc/pycbc_make_coinc_search_workflow

```diff
@@ -355,6 +349,19 @@ for inj_file, tag in zip(inj_files, inj_tags):
         file_for_injfind = optimal_snr_file
     else:
         file_for_injfind = inj_file
+
+    if workflow.cp.has_option_tags('workflow-injections', 'strip-injections', tags=[tag]):
+        veto_inj_file = wf.veto_injections(workflow, file_for_injfind, seg_summ_file,
```

Comment:
I think this line should fall back to using the "inj_file" (without the optimal SNRs) when you don't also use the injection cutting based on SNR. Then we can run in the default configuration with no change.